app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
I'm not sure here... According to the documentation errors raised in development shouldn't be captured by the error block unless we specify an `:after_handle` option to `show_exceptions`.  Maybe I'm interpreting things in the wrong way. Here is the quote:

> show_exceptions
> Show a stack trace in the browser when an exception happens. Enabled by default when environment is set to "development", disabled otherwise.
> Can also be set to :after_handler to trigger app-specified error handling before showing a stack trace in the browser.

My interpretation is that @Insti is right here in saying that this block won't get executed in dev.